etc/cron.d/bedrock-prod.tmpl

```diff
@@ -9,6 +9,7 @@ MAILTO="webops-cron@mozilla.com,cron-bedrock@mozilla.com"
 
 */10 * * * * {{ user }} {{ source }}/bin/update-scripts/prod/update-prod-php.sh {{ log }}
 */15 * * * * {{ user }} {{ source }}/bin/update-scripts/prod/update-prod-locale-cron.sh {{ log }}
+0    * * * * {{ user }} {{ source }}/bin/update-scripts/prod/update-prod-product-details.sh {{ log }}
```

Comment:
@sylvestre mentioned in irc that he would prefer that this update with the same frequency that we do the rnasync. Any objections to changing that first `0` to a `*/5` here and on the stage template?